src/content/release-notes.mdx

```diff
@@ -5,6 +5,48 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.19.0
+
+16 April 2024
+
+This versions is compatible with Kubernetes versions 1.25 to 1.29
```

Comment:
@ifbyol confirming we are officially supporting 1.29 in this release

Reply:
1.29 is on beta on this release, and 1.25 should be marked as deprecated

Reply:
AFAIK, we jsut did a change to start running tests in 1.29, but nothing else was modified. We show a warning in the installation, so I would say we don't support it. At least, not in the same way we did in the past

Reply:
just confirming. We should update this to be 1.26 to 1.28?

Reply:
@codyjlandstrom 1.19 supports 1.29 in beta, and marks 1.25 for deprecation on the next release, but 1.25 is till supported.
Our policy is:
- Run all tests for a new kubernetes release
- Mark the new kubernetes release as beta support, and mark for deprecation the old one
- For 1.20, support for 1.25 is removed, and 1.29 is fully supported and the recommended one

Reply:
> @ifbyol we have run all the tests in all the different cloud providers, and staging and the product cluster are running in 1.29 for a while now. @mnevadom could you check why we are showing a warning for 1.29 and incorporate this change to our guides to support a new kubernetes version?

Normally we do it in two steps, first add the version to test in nightlies and then modify the Chart and if the old version is already removed, remove it from support. 
My understanding from your messages is that we should already add for this version the warning including 1.29, and once we release/cut this version, remove 1.25 right?

Reply:
@mnevadom we support 1.29, we shouldn't show a warning for 1.129 anymore. And yes, once we release this chart version, support for 1.25 can be removed